config/src/test/kotlin/org/springframework/security/config/web/servlet/session/SessionFixationDslTests.kt

```diff
@@ -26,10 +26,10 @@ import org.springframework.mock.web.MockHttpSession
 import org.springframework.security.config.annotation.web.builders.HttpSecurity
```

Comment:
Please revert the changes to this file, since the only changes are import ordering.